alfred-telemetry-solr/README.md

```diff
@@ -105,77 +105,77 @@ At the moment following metrics are included, for each core tracked by solr:
-| solr_alfresco_searcher_maxDoc                                         |
-| solr_alfresco_searcher_numDocs                                        |
-| solr_alfresco_searcher_indexVersion                                   |
+| solr.alfresco./afts.handlerStart                                      |
+| solr.alfresco./afts.requests                                          |
+| solr.alfresco./afts.timeouts                                          |
+| solr.alfresco./afts.errors                                            |
+| solr.alfresco./afts.clientErrors                                      |
+| solr.alfresco./afts.serverErrors                                      |
+| solr.alfresco./afts.75thPcRequestTime                                 |
+| solr.alfresco./afts.95thPcRequestTime                                 |
+| solr.alfresco./afts.99thPcRequestTime                                 |
+| solr.alfresco./afts.999thPcRequestTime                                |
+| solr.alfresco./afts.medianRequestTime                                 |
+| solr.alfresco./afts.avgTimePerRequest                                 |
+| solr.alfresco./afts.totalTime                                         |
+| solr.alfresco./afts.5minRateRequestsPerSecond                         |
+| solr.alfresco./afts.15minRateRequestsPerSecond                        |
+| solr.alfresco./afts.avgRequestsPerSecond                              |
+| solr.alfresco./cmis.handlerStart                                      |
+| solr.alfresco./cmis.requests                                          |
+| solr.alfresco./cmis.timeouts                                          |
+| solr.alfresco./cmis.errors                                            |
+| solr.alfresco./cmis.clientErrors                                      |
+| solr.alfresco./cmis.serverErrors                                      |
+| solr.alfresco./cmis.75thPcRequestTime                                 |
+| solr.alfresco./cmis.95thPcRequestTime                                 |
+| solr.alfresco./cmis.999thPcRequestTime                                |
+| solr.alfresco./cmis.medianRequestTime                                 |
+| solr.alfresco./afts.avgTimePerRequest                                 |
+| solr.alfresco./cmis.totalTime                                         |
+| solr.alfresco./cmis.5minRateRequestsPerSecond                         |
+| solr.alfresco./cmis.15minRateRequestsPerSecond                        |
+| solr.alfresco./cmis.avgRequestsPerSecond                              |
+| solr.alfresco.searcher.warmupTime                                     |
+| solr.alfresco.searcher.deletedDocs                                    |
+| solr.alfresco.searcher.maxDoc                                         |
+| solr.alfresco.searcher.numDocs                                        |
+| solr.alfresco.searcher.indexVersion                                   |
 
 ### Tomcat jmx metrics
```

Comment:
this is for solr4 only ?
do we have them in `alfred-telemetry-platform` ?

Reply:
Are you talking and jmx beans in general?
These are solr metrics, available in jconsole for both solr4 and solr6.

Reply:
it says "tomcat" metrics, while solr6 runs jetty ?

Reply:
Indeed, the implementation is for tomcat.
Metrics are there in jconsole for solr6 as well. There is no implementation for jetty. 
I looked how this could be implemented, but did not reach a conclusion.